src/vm/dllimport.cpp

```diff
@@ -7047,7 +7055,6 @@ HINSTANCE NDirect::LoadLibraryModule(NDirectMethodDesc * pMD, LoadLibErrorTracke
     bool libNameIsRelativePath = Path::IsRelative(wszLibName);
     if (hmod == NULL)
     {
-#ifndef FEATURE_CORECLR
         // First checks if the method has DefaultDllImportSearchPathsAttribute. If method has the attribute
```

Comment:
@yizhang82 the host is not doing anything different from how CoreCLR hosting used to work from Day 1. Enabling this interop scenario is new and I am hoping you can define the overall interop experience. @schellap can provide inputs and based upon the discussion, the workitems can be defined if the current implementation does not work.

We should not merge this PR until we understand its implication across the board. Do you have a sense of what will happen, for the various attribute values, once this is enabled?